### DISCUSSION TITLE
Support multiple flows with the same origin and dest

### DISCUSSION THREAD

**Discussion by openaphid** [FlowmapBlue/FlowmapBlue#52] [Q&A] [answered]
First thanks for building a useful tool. I wonder if there is a workaround to support multiple flows with the same origin and dest.

The service currently throws an error upon such data.

**Comment by ilyabo** [comment]
It shouldn't be an error, but a warning. The app actually sums the counts for the flows up. 

But there is no way to show them separately at the moment. Could you elaborate why would you need that?

**Reply by openaphid** [reply]
Thanks for your reply. 
I was trying to visualize a dataset on logistics between cities, and I'd like to show the count data per month in separate lines between two cities.

**Comment by ilyabo** [comment, marked as answer]
I see. This is currently not possible. You can however add the temporal dimension as another column like [in this map](https://flowmap.blue/1Ekx_2rvuq-OnxTlPyFZ2ryQSphHG4KW2llDgzDoF-RA?v=57.679760%2C11.949905%2C10.68%2C0%2C0&a=0&as=1&b=1&bo=75&c=1&ca=1&d=1&fe=1&lt=1&lfm=ALL&t=19990101T000000%2C20200101T000000&col=Default&f=50), for example
